src/components/Questionnaire/QuestionTypes/MedicationStatementQuestion.tsx

```diff
@@ -114,7 +114,19 @@ const MEDICATION_STATEMENT_FIELDS: FieldDefinitions = {
     required: true,
     validate: (value: unknown) => {
       const period = value as { start?: string; end?: string };
-      return !!period?.start;
+      if (!period?.start) {
+        throw Error("start_date_required");
```

Comment:
apply `t` function from where it's being thrown itself instead of catches.

that way we could easily run static analysis tools to determine if text has been covered by translations or not.

```suggestion
        throw Error(t("start_date_required"));
```